### DISCUSSION TITLE
Using podman for armv7 32bit processors

### DISCUSSION THREAD

**Discussion by sparshstark** [containers/podman#22021] [Q&A] [answered]
Hey guys, I have to use podman to run containers on a 32-bit armv7 processor. Do we have a version release having the support for it?

**Comment by Luap99** [comment]
I am not sure what exactly you are asking, podman should work on armv7.

**Reply by sparshstark** [reply]
I was asking does it have a 32-bit processor support, because i need to create a container which supports a 32-bit arms processor on a 64-bit machine for some work.

**Reply by afbjorklund** [reply, marked as answer]
Assuming you have the proper qemu-user binaries installed, you can use `--platform=linux/arm/7` when building / running.

**Reply by Luap99** [reply]
If you want to emulate some other arch you have to install an emulator for that arch and register it via binfmt_misc, `qemu-user-static` is the most common one AFAIK.

**Reply by sparshstark** [reply]
Hey, thank you soo much to both of you !
